build_bundle_image.sh

```diff
@@ -346,7 +346,7 @@ function prepare_temp_directory {
 
 		if [ ! -e "${TEMP_DIR}/liferay/tomcat" ]
 		then
-			tomcat_dir_name="tomcat-${tomcat_version}"
+			mv "${TEMP_DIR}/liferay/tomcat-"* "${TEMP_DIR}/liferay/tomcat"
```

Comment:
@kiwm, try to use the `latest_tomcat_version` as the variable to download the newest Tomcat version instead. It'll simplify your fix.